gradle/libs.versions.toml

```diff
@@ -0,0 +1,42 @@
+[versions]
+forgeGradle = "6.0"
```

Comment:
I am not a fan of using toml for stuff like this, IF we were to do this then putting it in the settings.gradle would be the way to go as thats how I do it for all other projects in the org
https://github.com/MinecraftForge/Bootstrap/blob/master/settings.gradle#L12

However I've not bothered to do that here because there is no sub-project so there is no need to sync versions between them. So it's not really worth doing. Only real benefit would be the cases where we use the same version for multiple dependencies.

Reply:
Gradle notoriously 'recommends' many things, and then changes their mind every 3 months.
The settings.gradle is the same thing as the toml. They both use the same system just declared in a different location. 
Both are 'recommended' by gradle.

Reply:
They do not in fact use the same system, version catalogs are type safe. But since this project uses a Groovy buildscript that doesnt really matter anyways. I removed the catalog in my latest commit, consider it fixed ^^